344-reverse-string.js

```diff
@@ -5,9 +5,8 @@
 
 
 function reverseString(str) {
-	var newStr = str.split("").reverse().join("");
-	return newStr;
-};
+	return newStr = str.split("").reverse().join("");
```

Comment:
@meowbot24 Don't assign the `newStr` variable. Just do `return str.split("").reverse().join("");` 😸